client/flutter/lib/pages/main_pages/home_page.dart

```diff
@@ -1,15 +1,131 @@
 import 'package:flutter/cupertino.dart';
-import 'package:who_app/components/home_page_header.dart';
+import 'package:who_app/api/content/schema/fact_content.dart';
+import 'package:who_app/api/linking.dart';
+import 'package:who_app/components/home_page_sections/home_page_donate.dart';
+import 'package:who_app/components/home_page_sections/home_page_header.dart';
+import 'package:who_app/components/home_page_sections/home_page_information_card.dart';
+import 'package:who_app/components/home_page_sections/home_page_protect_yourself.dart';
+import 'package:who_app/components/home_page_sections/home_page_recent_numbers.dart';
 import 'package:who_app/components/page_scaffold/page_scaffold.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
 
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return PageScaffold(
       showHeader: false,
+      color: Constants.greyBackgroundColor,
       body: [
-        HomePageHeader(HeaderType.ProtectYourself),
+        _HomePageSection(
+          content: HomePageHeader(HeaderType.ProtectYourself),
+        ),
+        _HomePageSection(
+          padding: EdgeInsets.only(top: 56.0),
+          header: _HomePageSectionHeader(
+            // TODO: localize
+            title: 'Recent Numbers',
+            linkText: 'See all ›',
```

Comment:
Do we want the carets inside the link as opposed to as an icon?

Reply:
This is definitely a good question... in the design, it just uses the character in text. When it comes to localization, for RTL we'll want to reverse the caret, which may mean it makes more sense to keep this in the localized string, or might mean that having it in the string makes it more difficult... I think we can decide that in a later PR